File: conexiones/conexion.py
import mysql.connector

    # Intentamos establecer la conexion de base de datos
conn = mysql.connector.connect(
    host = "localhost",     # Direccion del servidor de la base de datos
    user = "root",          # Usuario de MySQL
    password = "",     # Contraseña del usuario MySQL (debe configurarse si tienes una)
    #database = "tienda", # Descomentar si se quiere conectar a una base de datos especifica.
)
    # Si la conexion se establece correctamente, imprimimos un mensaje de exito
print(conn)
print("Conexion exitosa")

File: conexiones/conexion.php
<?php
$host = "localhost";
$user = "root";
$password = "";
// $dbname = "tienda"; // Descomenta y ajusta si deseas conectar a una base de datos específica

$conn = new mysqli($host, $user, $password);

if ($conn->connect_error) {
    die("Conexión fallida: " . $conn->connect_error);
}

echo "Conexión exitosa\n";
$conn->close();
?>
